src/inputs/Checkbox/Checkbox.test.tsx

```diff
@@ -0,0 +1,43 @@
+import { h } from "preact";
+import { render, fireEvent } from "@testing-library/preact";
+import "@testing-library/jest-dom/extend-expect";
+import { Checkbox } from "./Checkbox";
+
+
+test("onChange handler should be called when clicked on it", () => {
+  const handleChange = jest.fn();
+
+  const { getByRole } = render(<Checkbox onChange={handleChange}>Some Label</Checkbox>);
+  const checkboxElement = getByRole("checkbox");
+
+  fireEvent.click(checkboxElement);
```

Comment:
**nitpick:** when working on inputs, we should use `fireEvent.change` to keep up with semantics